data_utils/raw_dataset_loader.py

```diff
@@ -68,18 +68,117 @@ def load_raw_data(self, dataset_path: str, config):
         ) = self.__normalize_dataset(dataset=dataset)
 
         serial_data_name = config["name"]
-        serial_data_path = (
-            os.environ["SERIALIZED_DATA_PATH"]
-            + "/serialized_dataset/"
-            + serial_data_name
-            + ".pkl"
-        )
+        if dataset_type == "total":
+            serial_data_path = (
+                os.environ["SERIALIZED_DATA_PATH"]
+                + "/serialized_dataset/"
+                + serial_data_name
+                + ".pkl"
+            )
+        else:
+            # append for train; test; validation
+            serial_data_path = (
```

Comment:
`serial_data_name = config["name"] + "_" + dataset_type+ ".pkl"`
Then create `serial_data_path` outside

Reply:
updated in e64f94a12323